components/css_selector.rst

```diff
@@ -47,11 +47,14 @@ The CssSelector Component
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The component's only goal is to convert CSS selectors to their XPath
-equivalents::
+equivalents, using :method:`Converter::toXPath() <Symfony\\Component\\CssSelector\\Converter::toXPath>`
```

Comment:
no need to use a special text for the link IMO. Just use ``:method:`Symfony\\Component\\CssSelector\\CssSelectorConverter::toXPath` ``